sdk/search/search-documents/review/search-documents-node.api.md

```diff
@@ -292,6 +219,7 @@ export interface BaseVectorSearchCompression {
     defaultOversampling?: number;
     kind: "scalarQuantization" | "binaryQuantization";
     rerankWithOriginalVectors?: boolean;
+    // Warning: (ae-forgotten-export) The symbol "RescoringOptions" needs to be exported by the entry point index.d.ts
```

Comment:
`ae-forgotten-export` should be treated as an error so we don't forget to use authored types.